pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -196,6 +198,8 @@ type CruiseControlConfig struct {
 	CruiseControlAnnotations map[string]string `json:"cruiseControlAnnotations,omitempty"`
 	// InitContainers add extra initContainers to CruiseControl pod
 	InitContainers []corev1.Container `json:"initContainers,omitempty"`
+	// ResourceRequirements for initContainers
+	InitContainersResourceRequirements *corev1.ResourceRequirements `json:"initContainersResourceRequirements,omitempty"`
```

Comment:
This field is redundant. The InitContainers []corev1.Container field already defines resource requirements for each init container.